Assignment2/Saabertooth1-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,48 @@
+{
+  "@context": {
+      "@type": "person",
+      "xsd": "http://www.w3.org/2001/XMLSchema",
+      "fullName": "http://example.org/fullname",
+      "age": "xsd:integer",
```

Comment:
Wrong datatype definition. You have to define the property of the vocabulary representing the age and then the specific datatype; e.g. "age":{"http://example.org/age","@type:":"xsd:integer"} Otherwise integer appears as the property.

Reply:
This example gave me a syntax error. I solved it by adding an @id like this "age": {"@id": "http://example.org/age","@type:":"xsd:integer"} I don't know if there is a less complex definition or if this is the correct way of doing it.

Reply:
Sorry, I forgot to add "@id".